decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -111,55 +118,48 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
         if self.validateSeed().valid {
             self.activateConfirmButton()
         } else {
-            self.deactivateConfirmButton()
+            self.deactivateConfirmButtonForError()
```

Comment:
Why is the function renamed? If the seed is not valid, deactivate confirm button, that seem straightforward to me. What is deactivate confirm button for error?

Reply:
the button is not  really deactivated it blocks it from proceeding and display the appropriate error message when clicked.